perceval/rendering/pdisplay.py

```diff
@@ -320,10 +326,88 @@ def pdisplay_tomography_chi(qpt: ProcessTomography, output_format: Format = Form
     plt.show()
 
 
+def _complex_to_rgb(z: complex, cmap='hsv'):
```

Comment:
Could you move your new utils fonctions to another python module (even a private one - i.e. prefix the py file name with an underscore)?